packages/@okta/i18n/src/properties/login.properties

```diff
@@ -42,7 +42,7 @@ error.unsupported.browser = Unsupported browser
 error.unsupported.cors = Unsupported browser - missing CORS support
 error.unsupported.localStorage = Unsupported browser - missing localStorage support
 error.enabled.cors = There was an error sending the request - have you enabled CORS?
-error.expired.session = Your session has expired. Please try to log in again.
+error.expired.session = Your session has expired. Please try to sign in again.
```

Comment:
Please let me know what you think of this change. Guidance from UX is to always say `sign in` not `log in`. This breaks some selenium tests. Will create another PR to fix those.

Reply:
This change is good, so we should keep it

With that said, you're adding a new controller just for mfa invalid session error, so this error might not be the best. The user can't really go and sign in again since we landed in a dead-end error page without links. Maybe we should have a different error message? Something generic like
_"Error validating MFA. Please try again"_ (not sign in specifically, but whatever it is they were trying)
What do you think?

Reply:
I think it's ok to keep the same message. mfa is also part of signing in. They would need to close the window / refresh to restart the flow. what do you think of that experience?

Reply:
@gowthamidommety-okta  when ever we update the text we need a new key.

Reply:
Also this text doesnt make sense in MFA only flow. What can the user do at this point just close the modal and retry ?

Reply:
why do we need a new key?

And I still would think it is better to have a different error for mfa only flows... can we double check with UX. If they are good then I will say no more

Reply:
I already checked with @vincentvoong-okta but let me check again.

Reply:
@gowthamidommety-okta @mauriciocastillosilva-okta 
Initially I thought we had to reuse an existing error message. It sounds like this proposal is going to be a new message - I think that is fine so long as this message only appears in an MFA dead-end state. I recommend "Unable to authenticate at this time. Try again.".